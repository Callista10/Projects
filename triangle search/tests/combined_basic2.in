-N=10 -mindist=1 -rseed=3