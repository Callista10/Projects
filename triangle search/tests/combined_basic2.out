read 10 points
-26.20, 4.42
-13.00, 10.39
2.32, 24.13
This is a triangle
